libbeat/docs/monitoring/monitoring-internal-collection.asciidoc

```diff
@@ -11,22 +11,38 @@
 
 [role="xpack"]
 [[monitoring-internal-collection]]
+
+
+[IMPORTANT]
```

Comment:
Content changes look good, but you need to move this note to a valid location. If you want it to appear at the top of the topic about internal collection, move it to line 19 (after the title abbrev and before the content).

Current location results in build errors. sorry, I always assume folks do a local build, so I didn't notice the invalid structure.